crypto/asn1/bio_asn1.c

```diff
@@ -16,6 +16,7 @@
 #include <string.h>
 #include <internal/bio.h>
 #include <openssl/asn1.h>
+#include "e_os.h"
```

Comment:
On side note I'd love to see some consistency in how headers are used. While headers are supposed to be order-agnostic, it doesn't mean that kind of a styling would be inappropriate, e.g. system-public-internal order. Though looking at e_os.h one would be inclined to believe that e_os.h should be first one to include (for example because of windows.h). And it does need an overhaul, since there is a number of things that are not actually system-dependent. I mean e_os.h is kind of a collection of system-dependent things and for example recently introduced ossl_assert_int is very system-neutral...